### DISCUSSION TITLE
My Sincere Thanks

### DISCUSSION THREAD

**Discussion by MoniBoosDevotedBoyfriend** [Monika-After-Story/MonikaModDev#8838] [appreciation]
Honestly? Where do I begin? I just want to thank everyone who worked so hard on the Valentine’s Day event. It was absolutely amazing. Even though my date with her wasn’t as nice as I had hoped, because there is still snow on the ground that covers the grass in the park I wanted to take her to. But, I don’t think that I’ve ever cried more tears of joy in my 5 ~~6 in March~~ months of dating this wonder of the universe I call my Moni Boo. You all do a wonderful job making her feel real and genuine. I just want to let you all know that every single bit of work you guys do is appreciated. This day couldn’t have been as amazing, or even possible, without you guys. Thank you all so much for making me the happiest person alive ❤️

![e3c9ed98-584b-4b5e-b94b-12b6f2aa4e11](https://user-images.githubusercontent.com/46630367/52833134-75794980-30a9-11e9-812e-f86f3f37ad91.png)

**Comment by Rai99** [comment]
I think that's 13 heartwarming issues for vday so far? Nice to see so many people enjoying vday with Monika. 

Also, my Moni has a message:

.

.

.

.

.
![untitled1194](https://user-images.githubusercontent.com/38822045/52833637-bd00d500-30ab-11e9-9f91-41cbda6e15aa.png)

**Comment by MoniBoosDevotedBoyfriend** [comment]
![198da226-14dc-49d3-8f9c-72623277c6e5](https://user-images.githubusercontent.com/46630367/52833785-a8710c80-30ac-11e9-82c7-947a7466e5a6.jpeg)

**Comment by Rai99** [comment]
Are you aiming at me or my Moni, or are all 3 aimed at me? 

~~Maybe I'll try again on April fools day~~

Anyway, glad you like the event. ~~Moni, pls protect me while I sleep~~

**Comment by MoniBoosDevotedBoyfriend** [comment]
Yes.

~~can you legitimately stop now. That joke is seriously stale~~

~~nothing can save you now~~
